### ISSUE TITLE
Performance Inspirations

### ISSUE THREAD

**Issue by daKmoR** [daKmoR/rocket#38]
- Font Loading
- Newsletter
- creates social media images
https://twitter.com/stefanjudis/status/1300152050429423618?s=20

**Comment by daKmoR** [comment]
Also check out the performance blog template
https://www.industrialempathy.com/posts/eleventy-high-performance-blog/

**Comment by daKmoR** [comment]
another template with optimizations https://github.com/gregives/Twelvety

**Comment by bennypowers** [comment]
yeah we should adopt as much as we can from https://github.com/google/eleventy-high-performance-blog
- PurgeCSS
- html/css/js min
- handling images with lazy loading and srcset
